### PR TITLE
style(settings): General style updates

### PR DESCRIPTION
**Pull request by vpomerleau**
## Because

* Switching design to text-start default instead of text-center
* Some images were cut off on mobile

## This pull request

* Update style for AppLayout
* Update image styling
* Various tweaks to accommodate change to text-start
* Set size for Mozilla logo in Storybook sidebar
* Replace placeholder image used in storybooks (placekitten server no longer seems to work)
* Adjust padding for data block, third party auth buttons
* Fix error banner shown instead of success banner on inline 2FA setup

## Issue that this pull request solves

Closes: #FXA-10613, FXA-10741

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Start align and image fix on mobile:

image

Signin (with start text align) - opted to leave email centered with avatar

image

Signup (sync) with text-start

image

## Other information (Optional)

Opted to keep content centered on a few select screens:

Legal:
image

Confirmation pages that only contained a heading, centered image and one-liner text or CTA. E.g.,

image

### CODE REVIEW

**Review by vbudhram — APPROVED** [review]
@vpomerleau Thanks! Storybook views LGTM 👍🏽

File: packages/fxa-react/styles/flows.css [hunk 1]
```diff
@@ -6,13 +6,8 @@
...
 /* Transparent border around card is for Windows HCM mode */
 
-/* Does not specify text-align */
-.card-base {
-  @apply relative w-full border border-transparent mobileLandscape:w-120 mobileLandscape:bg-white rounded-xl px-8 pb-12 pt-9 mobileLandscape:shadow-card-grey-drop mb-6;
-}
-
 .card {
-  @apply card-base text-center;
+  @apply relative w-full border border-transparent mobileLandscape:w-120 mobileLandscape:bg-white rounded-xl px-8 py-9 mobileLandscape:shadow-card-grey-drop mb-6;
```
Comment (LZoog) [inline packages/fxa-react/styles/flows.css:10]: Yaaaaas

File: packages/fxa-settings/src/pages/mocks.tsx [hunk 2]
```diff
@@ -25,9 +25,10 @@ export const MOCK_AUTHPW =
   'b5a61d1f7a6b1b762539bd85f783b65f635def1025c3f66fc51a438a68a77d6d';
 export const MOCK_UNBLOCK_CODE = 'A1B2C3D4';
 export const MOCK_RECOVERY_CODE = 'a1b2c3d4e5';
+export const PLACEHOLDER_IMAGE_URL = 'https://loremflickr.com/512/512';
```
Comment (LZoog) [inline packages/fxa-settings/src/pages/mocks.tsx:28]: Nice cleanup here too.

**Review by LZoog — APPROVED** [review]
I didn't check the updated changes but quick glance through the code changes LGTM!

File: packages/fxa-settings/src/pages/mocks.tsx (hunk 2 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/pages/mocks.tsx:28]: `placekitten` no longer works 😢

**Pull request merged**